specification/monitor/resource-manager/Microsoft.Insights/stable/2018-09-01/metricBaselines_API.json

```diff
@@ -0,0 +1,381 @@
+            "required": false
+          },
+          {
+            "$ref": "#/parameters/BaselineResultTypeParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Successful request to get the list of metric baseline values.",
+            "schema": {
+              "$ref": "#/definitions/MetricBaselinesResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ErrorResponse": {
+      "description": "Describes the format of Error response.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Error code",
+          "type": "string"
+        },
+        "message": {
+          "description": "Error message indicating why the operation failed.",
+          "type": "string"
+        }
+      }
+    },
+    "MetricBaselinesResponse": {
+      "type": "object",
+      "properties": {
+        "timespan": {
+          "type": "string",
+          "description": "The timespan for which the data was retrieved. Its value consists of two datetimes concatenated, separated by '/'.  This may be adjusted in the future and returned back from what was originally requested."
+        },
+        "interval": {
+          "type": "string",
+          "format": "duration",
+          "description": "The interval (window size) for which the metric data was returned in.  This may be adjusted in the future and returned back from what was originally requested.  This is not present if a metadata request was made."
+        },
+        "namespace": {
+          "type": "string",
+          "description": "The namespace of the metrics been queried."
+        },
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SingleMetricBaseline"
+          },
+          "description": "The baseline for each time series that was queried."
+        }
+      },
+      "required": [
+        "timespan",
+        "interval"
+      ],
+      "description": "A list of metric baselines."
+    },
+    "SingleMetricBaseline": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The metric baseline Id."
+        },
+        "type": {
+          "type": "string",
+          "description": "The resource type of the metric baseline resource."
+        },
+        "metricName": {
```

Comment:
For api version 2019-03-01, the property name is 'name'

https://github.com/Azure/azure-rest-api-specs/blob/521ed887e283fc4de186fa59c6b02cdbcd7f73fb/specification/monitor/resource-manager/Microsoft.Insights/stable/2019-03-01/metricBaselines_API.json#L145-L147

Reply:
Please make sure you want to rename it.

Reply:
checked.